### DISCUSSION TITLE
Sync error in PE 13

### DISCUSSION THREAD

**Discussion by sanjeevstunner** [ROM-builders/temporary#14856] [General]
Build log: https://cirrus-ci.com/task/5533509161844736

Trees: 
https://github.com/sanjeevstunner/device_xiaomi_vayu/tree/wip
https://github.com/sanjeevstunner/hardware_xiaomi/tree/13

**Comment by sanjeevstunner** [comment]
Ig the manifest is trying to clone from official devices org, and that messes with local manifest. Same kinda issue happened with me while building locally. I had to manually delete pixel.xml from repo/local_manifest directory 

Hope this info helps

**Comment by Apon77** [comment]
What to do to fix?

**Reply by sanjeevstunner** [reply]
If I had one, I'd have already told you

**Reply by sanjeevstunner** [reply]
Maybe after executing these lines
https://cirrus-ci.com/task/5533509161844736?logs=sync#L24-L26
remove all files under **.repo/local_manifests/** other than local manifests that we use, that could fix the issue. As you dont allow rm command in build script, you have to do it on your side for it to fix


to test if my theory is right, trigger a build with my branch and if it results in same clone error, check .repo/local_manifests/, there'll be a pixel.xml file tracking device_xiaomi_vayu, and the tree has aosp.dep, and that will mess with other repos like vendor

**Comment by Apon77** [comment]
![Screenshot_20221028-204551_Termux](https://user-images.githubusercontent.com/61594531/198657430-d88bce98-0187-486b-973b-ced784c8d3a6.png)
There is only local manifest file

**Reply by sanjeevstunner** [reply]
Can you start build with my build.sh?

**Reply by sanjeevstunner** [reply]
https://cirrus-ci.com/task/5304339303497728?logs=sync#L62

BTW its causing issues for others too

**Reply by sanjeevstunner** [reply]
https://github.com/ROM-builders/temporary/blob/vayu-PixelExperience-thirteen-sanjeevstunner/build_rom.sh
Here's the script

**Comment by Apon77** [comment]
Pe 13 Dir removed. It should be fixed

**Reply by sanjeevstunner** [reply]
thanks
